### PR TITLE
Add jochenehret to CAPI group

### PR DESCRIPTION
**Pull request by jochenehret**

### CODE REVIEW

**Comment by Gerg** [comment]
Hey @jochenehret, you will need [two approvals for this PR](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#requirements-2) from current Application Runtime Interfaces Approvers. Once you have those, I can merge this.

**Review by johha — APPROVED** [review]
👍

**Review by sethboyles — APPROVED** [review]

**Comment by sethboyles** [comment]
👍

**Review by Gerg — APPROVED** [review]

**Pull request merged**
